### PR TITLE
Update basic PhotoGimp Settings

### PR DESCRIPTION
**Pull request by Avlye**
- Add Alternative Texts for Images
- Utilize raw github content instead of indirectly rendering images.
- Simplify descriptions for general users
- Change GIMP directory in Windows
- Add Patch Notes sections
- Use System Language instead of en_US
- Add GitIgnore
- Delete Ignored Folders
- Improve Plugins Script
- Add Editor Config

### CODE REVIEW

**Review by felipepedrosa — APPROVED** [review]

**Comment by Avlye** [comment]
By the way, that Issue #11  (aka Resynthesizer doesn't work on Windows) is easy to fix, so including Resynthesizer Windows's binaries is enough.

But this project seems to be more oriented to Linux installation, so I'm not sure about including windows binaries.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "python.linting.pylintEnabled": true,
+    "python.linting.enabled": true,
+    "python.formatting.provider": "black",
+    "python.pythonPath": "c:\\Program Files\\GIMP 2\\bin\\python.exe"
```
Comment (Avlye) [inline .vscode/settings.json]: This isn't ideal for cross-platform development.

Comment (Avlye) [reply .vscode/settings.json]: So I decided to remove VSCode settings.

File: .gitignore [hunk 2]
```diff
@@ -3,4 +3,7 @@
 
 # Ignore Personal GIMP Files
 .var/app/org.gimp.GIMP/data
-.var/app/org.gimp.GIMP/current
\ No newline at end of file
+.var/app/org.gimp.GIMP/current
```
Comment (Avlye) [inline .gitignore:6]: These (data, current) folders are generated by Gimp on startup and every project created, that contains only files related to that user, so isn't necessary.

**Pull request merged**